packages/app/src/app/components/UploadProgress/elements.ts

```diff
@@ -1,38 +1,40 @@
-import styled, { keyframes } from 'styled-components';
-
+import styled, { css, keyframes } from 'styled-components';
 import delayInEffect from '@codesandbox/common/lib/utils/animation/delay-effect';
 import OpaqueLogo from 'app/components/OpaqueLogo';
 import LocalLogo from 'react-icons/lib/md/laptop';
 import Cube from './Cube';
 
 export const Container = styled.div`
-  position: relative;
-  background-color: ${props => props.theme.background};
-
-  text-align: center;
-
-  font-size: 1.125rem;
-  padding: 1rem 2rem;
-  font-weight: 500;
-  color: rgba(255, 255, 255, 0.8);
+  ${({ theme }) => css`
+    position: relative;
+    padding: 1rem 2rem;
+    background-color: ${theme.background};
+    color: rgba(255, 255, 255, 0.8);
+    font-size: 1.125rem;
+    font-weight: 500;
+    text-align: center;
+  `}
 `;
 
 export const DeployAnimationContainer = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  padding: 3rem;
-
-  ${({ deploying }) => deploying && delayInEffect(0, false)};
+  ${({ deploying }: { deploying: boolean }) => css`
```

Comment:
@Saeris I think it's better to extract the `DeployAnimationContainerProps` here, just to be consistent